src/features/user/ManageProjects/ProjectsContainer.tsx

```diff
@@ -66,6 +66,14 @@ export default function ProjectsContainer({}: any) {
             {t('manageProjects:addProject')}
           </button>
         </Link>
+        <Link href="/profile/payouts">
+          <button
+            className="primaryButton"
+            style={{ width: '175px', marginTop: '20px', marginLeft: '10px' }}
+          >
```

Comment:
avoid inline styles instead create a separate class for it

Reply:
done